testing/test_unit.py

```diff
@@ -70,6 +70,26 @@ def pytest_html_results_summary(prefix, summary, postfix, session):
     result.assert_outcomes(passed=1)
 
 
+def test_chdir(pytester):
+    pytester.makepyfile(
+        """
+        import pytest
+
+        @pytest.fixture
+        def changing_dir(tmp_path, monkeypatch):
+            monkeypatch.chdir(tmp_path)
+
+        def test_function(changing_dir):
+            pass
+    """
+    )
+    page = pytester.runpytest("--html", "reports/report.html")
```

Comment:
Something that I just thought about.

Could the fact that we're hardcoding a forward slash in the call here be why `resolve` fails for Windows? 🤔

Reply:
Oh interesting!
I just pushed a new commit with `resolve()` and a proper path to see what happens.

Reply:
Seems it's the same... :-/

Reply:
I reverted to `Path.cwd()`, it seems it's the only way to make it work on all platforms and all Python versions...